components/drops/CreateDrop.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <div class="drop-card border card-border-color">
+    <div class="svg-icons">
+      <img :src="createdDrop1" class="svg-one" />
+      <img :src="createdDrop2" class="svg-two" />
+      <img :src="Ellipse" class="ellipse" />
+      <img :src="createdDrop3" class="svg-three" />
+      <img :src="Vector" class="vector" />
+      <img :src="createdDrop4" class="svg-four" />
+    </div>
+    <div class="inner_card">
+      <p>Create Your Own</p>
+      <NeoButton
+        tag="a"
+        href="https://form.kodadot.xyz/drop-page-interest"
+        target="_blank"
+        variant="secondary"
+        class="mt-2 neo-button">
+        Drop Page
+      </NeoButton>
+    </div>
+  </div>
+</template>
+<script setup lang="ts">
+import createdDrop1 from '@/assets/drops/create_drop1.svg'
+import createdDrop2 from '@/assets/drops/create_drop2.svg'
+import createdDrop3 from '@/assets/drops/create_drop3.svg'
+import createdDrop4 from '@/assets/drops/create_drop4.svg'
+import Ellipse from '@/assets/drops/Ellipse_36.svg'
+import Vector from '@/assets/drops/Vector_24.svg'
+import { NeoButton } from '@kodadot1/brick'
+</script>
+<style scoped lang="scss">
+@import '@/styles/abstracts/variables';
+.drop-card {
+  width: 350px;
+  height: 141px;
```

Comment:
do you need to have fixed size?

Reply:
fixed here https://github.com/kodadot/nft-gallery/pull/6675/commits/d76f8903855ffae4f323748fbba0fbd454546835